test/e2e/framework.go

```diff
@@ -2481,6 +2481,57 @@ func (data *TestData) GetTransportInterface() (string, error) {
 	return "", fmt.Errorf("no interface was assigned with Node IP %s", nodeIP)
 }
 
+func (data *TestData) GetPodInterfaceName(nodeName string, podName string) (string, error) {
+	antreaPod, err := data.getAntreaPodOnNode(nodeName)
+	if err != nil {
+		return "", fmt.Errorf("failed to get Antrea Pod on Node %s: %v", nodeName, err)
+	}
+	cmd := []string{"ip", "-br", "addr", "show"}
+	stdout, stderr, err := data.RunCommandFromPod(antreaNamespace, antreaPod, agentContainerName, cmd)
+	if stdout == "" || stderr != "" || err != nil {
+		return "", fmt.Errorf("failed to show ip address, stdout: %s, stderr: %s, err: %v", stdout, stderr, err)
+	}
+
+	lines := strings.Split(strings.TrimSpace(stdout), "\n")
+	for _, line := range lines {
+		if strings.Contains(line, podName) {
+			fields := strings.Fields(line)
+			name, _, _ := strings.Cut(fields[0], "@")
+			return name, nil
+		}
+	}
+
+	return "", fmt.Errorf("no interface was assigned with Pod %s", podName)
+}
+
+func (data *TestData) GetInterfaceMTU(nodeName string, interfaceName string) (int, error) {
+	antreaPod, err := data.getAntreaPodOnNode(nodeName)
+	if err != nil {
+		return 0, fmt.Errorf("failed to get Antrea Pod on Node %s: %v", nodeName, err)
+	}
+	cmd := []string{"ip", "link", "show", interfaceName}
+	stdout, stderr, err := data.RunCommandFromPod(antreaNamespace, antreaPod, agentContainerName, cmd)
+	if stdout == "" || stderr != "" || err != nil {
+		return 0, fmt.Errorf("failed to get interface MTU, stdout: %s, stderr: %s, err: %v", stdout, stderr, err)
+	}
+
+	// Example stdout:
+	// 22: vxlan-29acf3@if5: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1450 qdisc noqueue master ovs-system state UP mode DEFAULT group default
+	// link/ether be:41:93:69:87:02 brd ff:ff:ff:ff:ff:ff link-netns cni-320ae61f-5e51-d123-0169-9f1807390500
+	fields := strings.Fields(strings.Split(strings.TrimSpace(stdout), "\n")[0])
```

Comment:
It might be simper to get the MTU from the system file directly: cat /sys/class/net/<interface_name>/mtu

Reply:
Agreed, done.